test/validation/api/pktio/pktio.c

```diff
@@ -48,6 +48,9 @@ static int num_ifaces;
     interface that just become up.*/
 static bool wait_for_network;
 
+/* Dummy global variable to avoid compiler optimizing out API calls */
+uint64_t odp_valid_test_pktio_u64;
```

Comment:
please rename it without odp_. And I would move it inside loop and make it volatile then declaring somewhere on the top.

Reply:
This is "static volatile uint64_t test_pktio_dummy_u64"  in v3